pom.xml

```diff
@@ -76,5 +76,10 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
It's number 3 on the list of common Java libs
http://blog.takipi.com/the-top-100-java-libraries-in-2016-after-analyzing-47251-dependencies/